client/src/modules/reports/generate/recovery_capacity/recovery_capacity.html

```diff
@@ -29,6 +29,11 @@ <h3 class="text-capitalize" translate>REPORT.RECOVERY_CAPACITY.TITLE</h3>
               limit-min-fiscal>
             </bh-date-interval>
 
+            <bh-currency-select
```

Comment:
I suggest moving this down to right before the [Submit] button.  I think the "include unposted values" option affects the content of the report, so should be grouped with the other ones.  However the currency option only affects the format of the data, so I would put it lower in the list.